src/core/client/ui/components/v3/CallOut/CallOut.css

```diff
@@ -12,10 +12,30 @@
   border-left: 4px solid;
 }
 
+.container {
+  display: flex;
+  flex-direction: row;
```

Comment:
Is there a reason the component wasn't a `<Flex />` instead using the CSS directly?